src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -60,58 +60,79 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
-    val ownerGroup = makeRawlsGroup("testwsOwners", Set(owner), Set.empty)
+    val ownerGroup = makeRawlsGroup(s"rawls ${wsName} OWNER", Set(userOwner), Set.empty)
+    val writerGroup = makeRawlsGroup(s"rawls ${wsName} WRITER", Set(userWriter), Set.empty)
+    val readerGroup = makeRawlsGroup(s"rawls ${wsName} READER", Set(userReader), Set.empty)
+
     val workspace = Workspace(wsName.namespace, wsName.name, "aWorkspaceId", "aBucket", DateTime.now, DateTime.now, "testUser", Map.empty, Map(
-      WorkspaceAccessLevels.Owner -> ownerGroup), isLocked = true )
+      WorkspaceAccessLevels.Owner -> ownerGroup,
+      WorkspaceAccessLevels.Write -> writerGroup,
+      WorkspaceAccessLevels.Read -> readerGroup), isLocked = true )
 
     override def save(txn:RawlsTransaction): Unit = {
-      authDAO.saveUser(owner, txn)
+      authDAO.saveUser(userOwner, txn)
+      authDAO.saveUser(userWriter, txn)
+      authDAO.saveUser(userReader, txn)
       authDAO.saveGroup(ownerGroup, txn)
+      authDAO.saveGroup(writerGroup, txn)
+      authDAO.saveGroup(readerGroup, txn)
       workspaceDAO.save(workspace, txn)
     }
   }
 
   class DefaultTestData() extends TestData {
     // setup workspace objects
-    val user = RawlsUser(userInfo)
-    val ownerGroup = makeRawlsGroup("testwsOwners", Set(user), Set.empty)
-    val writerGroup = makeRawlsGroup("testwsWriters", Set(user), Set.empty)
-
+    val userOwner = RawlsUser(UserInfo("owner-access", OAuth2BearerToken("token"), 123, "123456789876543212345"))
+    val userWriter = RawlsUser(UserInfo("writer-access", OAuth2BearerToken("token"), 123, "123456789876543212346"))
+    val userReader = RawlsUser(UserInfo("reader-access", OAuth2BearerToken("token"), 123, "123456789876543212347"))
     val wsName = WorkspaceName("myNamespace", "myWorkspace")
+    val ownerGroup = makeRawlsGroup(s"rawls ${wsName} OWNER", Set(userOwner), Set.empty)
+    val writerGroup = makeRawlsGroup(s"rawls ${wsName} WRITER", Set(userWriter), Set.empty)
+    val readerGroup = makeRawlsGroup(s"rawls ${wsName} READER", Set(userReader), Set.empty)
+
     val wsAttrs = Map(
       "string" -> AttributeString("yep, it's a string"),
       "number" -> AttributeNumber(10),
       "empty" -> AttributeEmptyList,
       "values" -> AttributeValueList(Seq(AttributeString("another string"), AttributeBoolean(true)))
     )
-    val workspace = Workspace(wsName.namespace, wsName.name, "aWorkspaceId", "aBucket", DateTime.now, DateTime.now, "testUser", wsAttrs, Map(
-      WorkspaceAccessLevels.Owner -> ownerGroup))
 
-    val workspaceWriter = Workspace(wsName.namespace, wsName.name + "2", "aWorkspaceId2", "aBucket", DateTime.now, DateTime.now, "testUser", Map.empty, Map(
-      WorkspaceAccessLevels.Write -> writerGroup))
+    val workspace2 = Workspace(wsName.namespace, wsName.name + "2", "aWorkspaceId2", "aBucket2", DateTime.now, DateTime.now, "testUser", wsAttrs, Map(
+      WorkspaceAccessLevels.Owner -> ownerGroup))
```

Comment:
Workspace 2 should not be owned by the Workspace 1 Owner Group.  This inits Workspace 2 in a weird state.